php/php.blade/src/org/netbeans/modules/php/blade/editor/HyperlinkProviderImpl.java

```diff
@@ -0,0 +1,206 @@
+
+        if (tokensq == null) {
+            return null;
+        }
+
+        Token<PHPTokenId> currentToken = tokensq.token();
+        int startOffset = tokensq.offset();
+
+        if (currentToken == null) {
+            return null;
+        }
+
+        String focusedText = currentToken.text().toString();
+
+        //2 char config are not that relevant
+        if (focusedText.length() < 5 || !EditorStringUtils.isQuotedString(focusedText)) {
+            return null;
+        }
+
+        identifiableText = focusedText.substring(1, focusedText.length() - 1);
+        PHPTokenId prevTokenId = null;
+
+        while (tokensq.movePrevious() && tokensq.offset() >= lineStart) {
+            Token<PHPTokenId> token = tokensq.token();
+            if (token == null) {
+                break;
+            }
+            String text = token.text().toString();
+            PHPTokenId id = token.id();
+
+            if (prevTokenId != null && id.equals(PHPTokenId.PHP_STRING)) {
+                methodName = text;
+                //tooltip text
+                switch (methodName) {
+                    case "view":
+                    case "make":
+                    case "render":
+                        FileObject currentFile = EditorUtils.getFileObjectFromDoc(doc);
+
+                        if (currentFile == null) {
+                            return null;
+                        }
+                        List<FileObject> includedFiles = BladePathUtils.findFileObjectsForBladeViewPath(currentFile, identifiableText);
+                        String viewPath = BladePathUtils.toBladeToProjectFilePath(identifiableText);
+
+                        for (FileObject includedFile : includedFiles) {
+                            goToFile = includedFile;
+                            tooltipText = "Blade Template File : <b>" + viewPath
+                                    + "</b><br><br><i style='margin-left:20px;'>" + identifiableText + "</i>";
+                            goToOffset = 0;
+                            break;
+                        }
+
+                        return new int[]{startOffset, startOffset + currentToken.length()};
+                    default:
+                        return null;
+                }
+            }
+
+            if (id.equals(PHPTokenId.PHP_TOKEN) && text.equals("(")) {
+                prevTokenId = id;
+            }
+        }
+        return null;
+    }
+
+    @Override
+    public void performClickAction(Document doc, int offset, HyperlinkType type) {
+        switch (type) {
+            case GO_TO_DECLARATION:
+                switch (methodName) {
+                    case "view": // NOI18N
+                    case "make": // NOI18N
+                    case "render": // NOI18N
+                        if (goToFile != null) {
+                            openDocument(goToFile, goToOffset);
+                        }
+                        break;
```

Comment:
If you would like to use switch statements/other than enums with switch expressions, you should add `default` because we are unsure whether you didn't write it intentionally.
```java
default:
    // no-op
    break;
```